core/mobile/src/backend/MobileRpcServer.ts

```diff
@@ -102,6 +103,20 @@ export class MobileRpcServer {
     MobileRpcServer.interop.sendBinary = sender;
   }
 
+  private _sendPending() {
+    if (this._pendingMessages === undefined)
+      return;
+
+    for (const message of this._pendingMessages) {
+      this._connection!.send(message, (err) => {
+        if (err) {
+          throw err;
+        }
+      });
+    }
+    this._pendingMessages.length = 0;
```

Comment:
I feel that this would be better:
```ts
delete this._pendingMessages
```

Reply:
Does the original clear the outer pendingMessages? If so, then my suggestion above is obviously wrong.

Reply:
Why is that better? I just want to clear the array as these messages have been processed. The array itself gets reused the next time the app goes to the background.

Reply:
Sorry, my browser didn't seem to be updated when I responded. Yes, setting the length clears the outer pendingMessages array. The constructor just has a reference to that array.